index.d.ts

```diff
@@ -62,12 +63,14 @@ export interface CountUpProps extends CommonProps, CallbackProps {
   redraw?: boolean;
   preserveValue?: boolean;
   children?: (props: RenderCounterProps) => JSX.Element;
+  style?: React.CSSProperties;
```

Comment:
style, classname & children can be omitted I think.

Reply:
@glennreyes TS won't be happy if they will be omitted